github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```
@@ -613,8 +613,8 @@ postsubmits:
         - name: pgp-bot-key
           secret:
             secretName: pgp-bot-key
-    - name: post-project-infra-upload-testgrid-config
-      run_if_changed: '^(github/ci/prow-deploy/files/jobs/.*)|(github/ci/testgrid/gen-config\.yaml)$'
+    - name: post-project-infra-update-testgrid-config
+      run_if_changed: '^github/ci/prow-deploy/files/orgs\.yaml$'
```

This change for `run_if_changed` doesn't look right? Shouldn't it be like this?
```suggestion
      run_if_changed: '^github/ci/prow-deploy/files/jobs/.*$'
```

---

Absolutely, great catch, thanks!